network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -295,24 +289,24 @@ def get_missing_related_posts(self):
         """
         additional_posts = list()
         post_count = self.related_posts.all().count()
-        missing_count = self.related_post_count - post_count
+        missing_count = self.RELATED_POSTS_MAX - post_count
 
-        if missing_count == 0:
```

Comment:
This was failing when we had more related posts than max. While it is prevented in the panel it would break if we ever changed the max.